templates/output.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+		integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+	<!-- <link rel="stylesheet" type="text/css" href="css/style.css"> -->
+	<!-- <link rel="stylesheet" href="/static/style.css"> -->
+	<!-- <link rel= "stylesheet" type= "text/css" href= "url_for('static',filename='styles/mainpage.css') }}"> -->
+	<link rel="stylesheet" type="text/css" href={{ url_for('static', filename='css/style.css' ) }}>
+
+</head>
+
+<body style="background-color: #f4f4f4;" id="output-page">
+	<nav class="navbar" id="nav_output">
+		<div class="logo">
+			<button class="logo-nearby" type="button"><a class="nearby_button" href="#">Nearby</a></button>
+		</div>
+	</nav>
+	<main>
+		<!-- <h1>Organisations nearby</h1> -->
+		<form class="homepage">
+			<form action=" {{ url_for('out') }} ", method="GET">
+			<div class="container">
+			<div class="row justify-content-md-center">
+				<div class="col-4">
+					<label>Search</label>
+					<input type="search" class="form-control" placeholder="Search for social services...", name="search">
+				</div>
+				<div class="col-3">
+						<label for="bezirk">Search by neighborhood</label>
```

Comment:
The `for` attribute links the `label` based on the `id` of the form input, not the `name`, therefore we need to add the proper `id` value to the form input, right now its value is just `#`. The problem occurs to all labels.
It's a bit confusing but `id` differs from the `name`, `name` is used to send data when submitting the form.